eng/Versions.props

```diff
@@ -114,7 +114,7 @@
     <MicrosoftDiaSymReaderNativeVersion>17.0.0-beta1.21524.1</MicrosoftDiaSymReaderNativeVersion>
     <MicrosoftDiaSymReaderPortablePdbVersion>1.7.0-beta-21528-01</MicrosoftDiaSymReaderPortablePdbVersion>
     <MicrosoftExtensionsLoggingVersion>7.0.0</MicrosoftExtensionsLoggingVersion>
-    <MicrosoftExtensionsLoggingAbstractionsVersion>7.0.0</MicrosoftExtensionsLoggingAbstractionsVersion>
+    <MicrosoftExtensionsLoggingAbstractionsVersion>8.0.0</MicrosoftExtensionsLoggingAbstractionsVersion>
```

Comment:
These version changes should still be reflected in the Version.Details.xml file.

Reply:
I guess you mean the added pacakges?

Reply:
I meant all the corresponding dependencies whose versions you updated in this file. For example, this updates `MicrosoftExtensionsLoggingAbstractionsVersion` to `8.0.0` but `Microsoft.Extensions.Logging.Abstractions` is still set to `7.0.0` in the Version.Details.xml file.

As for missing entries in Version.Details.xml, there should at least be an entry defined there for each dependency that you're adding to SourceBuildPrebuiltBaseline.xml.

Reply:
Updated

Reply:
@mthalman  I do this in this pattern.
If the package is in `Versions.Details.xml`, I update the version.
If not, I check if it is in `https://github.com/dotnet/source-build-reference-packages`. If it in, it just get dimissed.
If it's not in `Version.Details.xml` and not added in source build reference packages repo. I put it in prebuild baseline.

I will leave this comment open. Let me know if this config file looks good to you